web/js/Controller.js

```diff
@@ -0,0 +1,143 @@
+class Controller {
+    constructor(main) {
+        /**
+         * Controller object based on MCV design pattern
+         * manages model (data) and view (visualizations) 
+         */
+        
+        // Controller Properties
+        this.selectedExecution = [1];
+        this.date = new Date();
+        this.SECOND = 1000 // ms
+        this.frameInterval = this.SECOND * 0.5
+        
+        // Initialize View components
```

Comment:
Only the stream view and history view will be inspected and tested with the current back-end for now.